examples/hosts/hosts-sample/src/app.ts

```diff
@@ -107,7 +108,8 @@ async function start() {
     }
 
     // Wait for connection so that proposals can be sent.
-    if (container !== undefined && !container.connected) {
+    // TODO: Remove null check after next release #8523
+    if (container !== undefined && container.connected !== undefined && !container.connected) {
```

Comment:
Not related to your change, but we should probably throw if container is still undefined.  As-is this will blow up in `getFluidObjectAndRender` which doesn't expect an `undefined` container.